vmslave-config/ansible/roles/install-packer-build-stack/tasks/main.yml

```diff
@@ -0,0 +1,67 @@
+- name: Check if packer is installed
+  shell: |
+    packer version
+  ignore_errors: yes
+  register: packer_check
+
+- name: Install packer
+  become: true
+  unarchive:
+    src: "{{ packer_zip }}"
+    dest: /usr/local/bin
+    remote_src: True
+  when: packer_check.rc != 0
+
+- name: Check if vmware workstation is installed
```

Comment:
is this step just for debug purpose ?
or if vmware workstation installed, will you skip the below download steps ?